tests/content/entity/procedural/test.js

```diff
@@ -20,7 +19,7 @@ autoTester.perform("Procedural create", Script.resolvePath("."), "secondary", fu
         lifetime: LIFETIME,
         userData: JSON.stringify({
             "ProceduralEntity": {
-                "shaderUrl": "https://github.com/" + user + repository + "blob/" + branch + "assets/shaders/proceduralShapeTestPosition.fs?raw=true",
+                "shaderUrl": assetsRepo + "assets/shaders/proceduralShapeTestPosition.fs",
```

Comment:
from what i said above, this could become assetsRepo + "shaders/proceduralShapeTestPosition.fs",